src/main/java/CommandManager.java

```diff
@@ -0,0 +1,75 @@
+import java.util.Scanner;
+
+public class CommandManager {
+    public static final String COMMAND_DONE = "done";
+    private static final Scanner in = new Scanner(System.in);
+    private static final String COMMAND_BYE = "bye";
+    private static final String COMMAND_DELETE = "delete";
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+    private static String userInput;
+
+    private static void setUserInput() {
+        userInput = in.nextLine().trim();
+    }
+
+
+    public static void executeCommand() {
+
```

Comment:
This method is too long (>30 LoC). You have abstracted out code into functions like `setUserInput()` and [print Divider()](url). How about taking abstraction further to `help` print statements? This would be good use of [SLAP](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard).